File: DSA/Assignment_3/row_change.c
#include <stdio.h>
int main()
{
    int m,n,r1,r2;
    printf("Enter row and column size:");
    scanf("%d %d",&m,&n);
    int arr[m][n];
    for(int i=0;i<m;i++)
    {
        for(int j=0;j<n;j++)
        {
            printf("ENetr element:");
            scanf("%d",&arr[i][j]);
        }
    }

    printf("Original matrix:\n");
    for(int i=0;i<m;i++)
    {
        for(int j=0;j<n;j++)
        {
            printf("%d ",arr[i][j]);
        }
        printf("\n");
    }
    
    printf("Enter rows to interchange:");
    scanf("%d %d",&r1,&r2);
    for(int j=0;j<n;j++)
    {
        int temp=arr[r1][j];
        arr[r1][j]=arr[r2][j];
        arr[r2][j]=temp;
    }

    printf("Modifiend matrix:\n");
    for(int i=0;i<m;i++)
    {
        for(int j=0;j<n;j++)
        {
            printf("%d ",arr[i][j]);
        }
    }
    
    return 0;
}

File: DSA/Assignment_3/row_change.py
m=int(input("Enter row size:"))
n=int(input("Enter column size:"))
arr=[]
for i in range(m):
    a=[]
    for j in range(n):
        a.append(int(input("Enter element:")))
    arr.append(a)

print("Original Matrix:")
for i in arr:
    print(i)

r1,r2=int(input("Enter row to interchange:")).split(' ')
for i in range(n):
    arr[r1][i],arr[r2][i]=arr[r2][i],arr[r1][i]
    